integration/ethereummock/node.go

```diff
@@ -351,7 +353,7 @@ func (m *Node) startMining() {
 
 			// Generate a random number, and wait for that number of ms. Equivalent to PoW
 			// Include all rollups received during this period.
-			common.ScheduleInterrupt(m.cfg.PowTime(), interrupt, func() {
+			async.ScheduleInterrupt(m.cfg.PowTime(), interrupt, func() {
```

Comment:
This can be replaced by `Schedule` since the interrupt is checked inside the function